New-PSPowerHour.ps1

```diff
@@ -0,0 +1,39 @@
+<#
+    .SYNOPSIS
+        Generate the markdown table of the presenters for the PSPowerHour
+    .NOTES
+        This utilizes PowerShellForGitHub module by the PowerShell team
+        For best results and the way written you will need a token from GitHub (no permissions needed to be assigned)
+#>
+[cmdletbinding()]
+param(
+    [string]$GitHubToken
+)
+
+if (-not (Get-Module PowerShellForGitHub)) {
```

Comment:
Should this also use the `ListAvailable` parameter as in the second example? Could this be in a loop to reduce duplicate code?